Applications/Console/src/main/java/com/agentdid127/resourcepack/Main.java

```diff
@@ -36,17 +35,8 @@ public static void main(String[] args) throws IOException {
 
         if (Util.getVersionProtocol(gson, from) > Util.getVersionProtocol(gson, to)) {
             new BackwardsPackConverter(from, to, light, minify, optionSet.valueOf(Options.INPUT_DIR), true, out).runDir();
-        }
```

Comment:
The else statement needs to be here to go backwards

Reply:
I didn't delete it?

Reply:
next line after this